### ISSUE TITLE
Host Clone COSMOS File/Directory Casing Issue

### ISSUE THREAD

**Issue by jlucas9** [nasa/nos3#127]
After creating the base box via `vagrant up`, if you then clone another version of NOS3 into the VM you run into an issue with COSMOS:

```
nos3@nos3:~/Downloads/nos3$ make launch 
./gsw/scripts/launch.sh
Make data folders...
42...
Simulators...
COSMOS Ground Station...
2023/02/08 17:54:15.094  FATAL: Fatal Exception! Exiting...
2023/02/08 17:54:15.094  FATAL: Error : Target folder must exist '/home/nos3/Downloads/nos3/gsw/cosmos/config/targets/../../../../COMPONENTS/ARDUCAM/GSW/ARDUCAM'.

Parsed output in /home/nos3/Downloads/nos3/gsw/cosmos/outputs/tmp/config/system/nos3_system.txt
/var/lib/gems/2.5.0/gems/cosmos-4.5.2/lib/cosmos/system/system.rb:474:in `block in process_targets'
```

**Comment by jlucas9** [comment]
Could be resolved as part of issue #175

**Comment by jlucas9** [comment]
Resolved as part of #163
